### PR TITLE
Fix straight quotes into apostrophes

### PR DESCRIPTION
**Pull request by kaelig**
What do you do when you have a few spare minutes while waiting for folks to join a meeting?

```
ag "[a-zA-Z]'[a-zA-Z]"

-> fix occurrences of ' into ’
```

[What is `ag`?](https://geoff.greer.fm/ag/)

### Why?

https://polaris.shopify.com/content/grammar-and-mechanics#section-punctuation

screenshot 2018-11-27 10 46 07

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
     "DiscardConfirmationModal": {
       "title": "Discard all unsaved changes",
-      "message": "If you discard changes, you'll delete any edits you made since you last saved.",
+      "message": "If you discard changes, you’ll delete any edits you made since you last saved.",
```
Comment (kaelig) [inline src/locales/en.json]: Will this break anything, or is it something we can change?

**Comment by kaelig** [comment]
Does this require a changelog entry?

**Comment by perrupa** [comment]
> Does this require a changelog entry?

I can't imagine any case in which this would break something or anyone caring about this. I'd error on the side of no changelog entry required. 👍

**Comment by kaelig** [comment]
Thank you @perrupa and @danrosenthal!

**Pull request merged**